### PR TITLE
blog post about Flux at KubeCon EU 2022

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-933--fluxcd.netlify.app/blog/

**Comment by dholbach** [comment]
Waiting on final Flex Your Flux image - until then any other reviews are appreciated.

File: content/en/blog/2022-05-12-flux-at-kubecon-eu-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+   working at a large insurance institution and helped them
+   transition to Flux and GitOps. Since then she has been a recurring
+   guest at Weave Online User Group, [GitOps
+   Days](https://www.gitopsdays.com/) and elsewhere.
+- **Stefan Prodan** is the creator of Flagger and has been a core
+   maintainer of Flux since Weaveworks donated the project to CNCF.
+   He is a maintainer of SMI and he\'s been part of Team Flux for 4+
+   years already. He is also the owner of a very cute dog.
+- **Scott Rigby** is maintainer of Flux and Helm, and one of the
+   chairs of Open GitOps. He knows the ecosystem very well,
```
Comment (scottrigby) [inline content/en/blog/2022-05-12-flux-at-kubecon-eu-2022/index.md:90]:
```suggestion
   chairs of OpenGitOps. He knows the ecosystem very well,
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
A few small changes. Otherwise looks great! Thanks for putting this together @dholbach 🦄✨

**Review by scottrigby — APPROVED** [review]
🚀

**Pull request merged**